hrim_qa/CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(hrim_qa)
+find_package(ament_cmake REQUIRED)
+
+install(
+  PROGRAMS
+  scripts/publisher_py.py
+  scripts/subscriber_py.py
+  scripts/requester_py.py
+  scripts/replier_py.py
+  DESTINATION lib/${PROJECT_NAME}
+)
+
+find_package(ros_testing REQUIRED)
+
+MACRO(SUBDIRLIST result curdir)
+  FILE(GLOB children RELATIVE ${curdir} ${curdir}/*)
+  SET(dirlist "")
+  FOREACH(child ${children})
+    IF(IS_DIRECTORY ${curdir}/${child})
+      LIST(APPEND dirlist ${child})
+    ENDIF()
+  ENDFOREACH()
+  SET(${result} ${dirlist})
+ENDMACRO()
+##############################################################################
+SET(DIR_HRIM /home/erle/ros2_mara_ws/src/HRIM/generated2)
```

Comment:
Need to fix this. To use a variable to introduce in the cmake

HRIM_DIRECTORY

--cmake-args -DHRIM_DIRECTORY=